src/main/java/com/meraphorce/dto/BulkResultDTO.java

```diff
@@ -0,0 +1,24 @@
+package com.meraphorce.dto;
+
+import java.util.List;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+/**
+ * Class to encapuslate the results for a Bulk process
+ * @author Juan Chavez
+ * @since May/18/2024
+ */
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@Getter
+@Setter
+public class BulkResultDTO {
```

Comment:
In this DTO you can be clearer instead of using the wild card ? .If you want to be more generic
public class BulkResultDTO<S,F> {
	private List<S> success;
	private List<F> failed;
}
 
But that could be useful when we have a lot of bulk operations, but is a good approach if you want to use it
An alternative is to use this way, is simpler.But is not that flexible

public class BulkUserResultDTO{
	private List<UserDTO> success;
	private List<UserDTO> failed;
}